src/h_vialib/secure/secrets.py

```diff
@@ -0,0 +1,49 @@
+import base64
+import json
+from typing import Tuple
+
+from Cryptodome import Random
+from Cryptodome.Cipher import AES
+from Cryptodome.Util.Padding import pad
+
+
+class SecureSecrets:
+    def __init__(self, secret: bytes):
+        self._secret = pad(secret, 16)
+
+    def encrypt_dict(self, payload: dict) -> str:
```

Comment:
Encrypting the dict (converting it to json etc) seems useful to have here rather that just expose low lever primitives.